flake.nix

```diff
@@ -2,6 +2,7 @@
   description = "NgiPkgs";
 
   inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+  inputs.php-nixpkgs.url = "github:drupol/nixpkgs?ref=php/add-new-builder";
```

Comment:
this is the fork/branch for https://github.com/NixOS/nixpkgs/pull/225401. Ideally once that PR is merged, we can just use the same `nixpkgs` as everything else and remove this `php-nixpkgs` param